### PR TITLE
Use correct bolus values in CBF calculation and temporarily disable Q2TIPS, multi-PLD, and no-BolusCutOff processing

### LINKED ISSUE #167: LabelingDuration not defined for PASL acquisitions
Hi, thanks for the excellent BIDS app.

I've been having issues processing some PASL data. [This line](https://github.com/PennLINC/aslprep/blob/6d125aee37614cce23032c8ae55e95e4f4acbc73/aslprep/workflows/asl/cbf.py#L99) which computes the "inflow times" (`--tis`) for BASIL seems to be configured to work only for (P)CASL data.

`LabelDuration` is a required field for (P)CASL data ([spec](https://bids-specification.readthedocs.io/en/v1.6.0/04-modality-specific-files/01-magnetic-resonance-imaging-data.html#pcasl-specific-metadata-fields)), but not for PASL data ([spec](https://bids-specification.readthedocs.io/en/v1.6.0/04-modality-specific-files/01-magnetic-resonance-imaging-data.html#pasl-specific-metadata-fields)).

`tiscbf` is passed right to BASIL, and `PostLabelingDelay` (as defined in the [BIDS spec](https://bids-specification.readthedocs.io/en/v1.6.0/04-modality-specific-files/01-magnetic-resonance-imaging-data.html#common-metadata-fields-applicable-to-both-pcasl-and-pasl)) seems to agree with the "inflow time" (`--tis`) that [BASIL/OxfordASL expects](https://asl-docs.readthedocs.io/en/latest/analysis_guide.html#post-label-delay-s). So, as a potential solution, perhaps the line [mentioned above](https://github.com/PennLINC/aslprep/blob/6d125aee37614cce23032c8ae55e95e4f4acbc73/aslprep/workflows/asl/cbf.py#L99):

```python
tiscbf=np.add(metadata["PostLabelingDelay"],metadata["LabelingDuration"])
```

could be made conditional on the `ArterialSpinLabelingType` field?

```python
tiscbf = get_tis(metadata)

# ...

def get_tis(metadata):
    if "CASL" in metadata["ArterialSpinLabelingType"]:
        return np.add(metadata["PostLabelingDelay"], metadata["LabelingDuration"])
    else:
        return np.array(metadata["PostLabelingDelay"])
```

Currently, the workaround I've been using is to set `"LabelingDuration": 0` in the sidecar, but this seems not ideal because the dataset fails BIDS validation and ASLPrep then has to be run with the `--skip-bids-validation` flag.

### PR DESCRIPTION
**Pull request by tsalo**
Closes #167.

## Changes proposed in this pull request
- For (P)CASL data, use `LabelingDuration` as the bolus value in BASIL CBF calculation. For PASL data, use `BolusCutOffDelayTime` instead.
- Apply the same fix to the vanilla CBF calculation.
- Raise an error if PASL data do not have BolusCutOffFlag set to True.
- Hopefully support QUIPSS PASL data as well.
- Explicitly block multi-PLD and Q2TIPS data for now.

I still need to figure out what to do if BolusCutOffFlag is False, as well as if BolusCutOffTechnique is QUIPSS or Q2TIPS.

### CODE REVIEW

File: aslprep/utils/misc.py [hunk 1]
```diff
@@ -234,11 +234,19 @@ def _prefix(subid):
...
 def pcasl_or_pasl(metadata):
     """Determine if metadata indicates a PCASL or ASL scan."""
-    if "CASL" in metadata["ArterialSpinLabelingType"]:
-        pcasl1 = True
-    elif "PASL" in metadata["ArterialSpinLabelingType"]:
-        pcasl1 = False
-    return pcasl1
+    aslt = metadata["ArterialSpinLabelingType"]
+
+    if "CASL" in aslt:
```
Comment (andrewrosss) [inline aslprep/utils/misc.py:239]: nit: I think it would be better to have this be `if aslt in ["CASL", "PCASL"]:`

**Comment by andrewrosss** [comment]
I'm not a core team member, but I'm the one who opened #167. These changes seem reasonable to me, specifically, using `LabelingDuration` for (P)CASL and `BolusCutOffDelayTime` for PASL seems to adhere to the intent of the BIDS spec and would fix address the original issue.

File: aslprep/utils/misc.py [hunk 2]
```diff
@@ -232,13 +232,22 @@ def _prefix(subid):
...
+def pcasl_or_pasl(metadata):
     """Determine if metadata indicates a PCASL or ASL scan."""
-    if "CASL" in metadata["ArterialSpinLabelingType"]:
-        pcasl1 = True
-    elif "PASL" in metadata["ArterialSpinLabelingType"]:
-        pcasl1 = False
-    return pcasl1
+    aslt = metadata["ArterialSpinLabelingType"]
+
+    if "CASL" in aslt:
```
Comment (andrewrosss) [inline aslprep/utils/misc.py]: nit: I think it would be better to have this be `if aslt in ["CASL", "PCASL"]:`

Comment (tsalo) [reply aslprep/utils/misc.py]: That's a good point. Thanks!

**Comment by tsalo** [comment]
Thank you for reviewing @andrewrosss. Your input is very much appreciated.

**Comment by andrewrosss** [comment]
Happy to help! :) Thanks for the fix!

**Pull request merged**